src/devices/schneider_electric.ts

```diff
@@ -1028,26 +1028,17 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: ['FLS/SYSTEM-M/4'],
-        model: 'WDE002906',
+        model: 'WDE002906/MEG5001-0300',
         vendor: 'Schneider Electric',
         description: 'Wiser wireless switch 1-gang or 2-gang',
         fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery],
         toZigbee: [],
-        endpoint: (device) => {
-            return {'right': 21, 'left': 22};
-        },
-        meta: {multiEndpoint: true},
-        exposes: [e.action(['on_left', 'off_left', 'on_right', 'off_right', 'brightness_move_up_left', 'brightness_stop_left',
-            'brightness_move_down_left', 'brightness_stop_left', 'brightness_move_up_right', 'brightness_stop_right',
-            'brightness_move_down_right', 'brightness_stop_right']), e.battery()],
-        configure: async (device, coordinatorEndpoint, logger) => {
-            // When in 2-gang operation mode, unit operates out of endpoints 21 and 22, otherwise just 21
-            const leftButtonsEndpoint = device.getEndpoint(21);
-            await reporting.bind(leftButtonsEndpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl', 'genPowerCfg']);
-            const rightButtonsEndpoint = device.getEndpoint(22);
-            await reporting.bind(rightButtonsEndpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
-            await reporting.batteryPercentageRemaining(leftButtonsEndpoint);
-        },
+        extend: [
+            deviceEndpoints({endpoints: {'right': 21, 'left': 22}}),
```

Comment:
Also add `battery` here

Reply:
it was already there i rearrange it for better readability